api/lib/stores/block-store.js

```diff
@@ -264,6 +266,68 @@ class BlockStore {
       'operatorAddress'
     )
   }
+
+  async updateNetworkInDB() {
```

Comment:
why are we updating all the networks individually? can't we update all of them at once?

Reply:
In store they are coming one at a time. With its validators and so on. I just update them as they come

Reply:
```suggestion
  async updateNetworkFromDB() {
```